docs/generating-code-from-schema.md

```diff
@@ -8,19 +8,22 @@ The plugin generates the following:
 
 ## Quick Start
 
+Code generation is typically integrated in the build.
+A Gradle plugin has always been available, and recently a Maven plugin was made [available](https://github.com/deweyjose/graphqlcodegen) by the community.
+
 To apply the plugin, update your project’s `build.gradle` file to include the following:
 ```groovy
 // Using plugins DSL
 plugins {
-	id "com.netflix.dgs.codegen" version "4.0.10"
+	id "com.netflix.dgs.codegen" version "4.2.0"
 }
 ```
 
 Alternatively, you can set up classpath dependencies in your buildscript:
 ```groovy
 buildscript {
    dependencies{
-      classpath 'com.netflix.graphql.dgs.codegen:graphql-dgs-codegen-gradle:latest.release'
+      classpath 'com.netflix.graphql.dgs.codegen:graphql-dgs-codegen-gradle:4.2.0'
```

Comment:
Add a disclaimer on using the latest version? the version here is used as example.
We could also use `4.2.+`?

Reply:
This doesn't work for plugins - only for normal dependencies

Reply:
Sorry, thanks for the clarification.

Reply:
I want to look into automating updating the versions in the docs each time we release, maybe based on a github action.
That's a todo though

Reply:
I'll make it 4.+, I think that's the best we can currently do?